types/params_test.go

```diff
@@ -9,10 +9,10 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 )
 
-func newConsensusParams(txsBytes, partSize int) ConsensusParams {
+func newConsensusParams(txsBytes, evidenceAge int) ConsensusParams {
```

Comment:
- txBytes should be called maxBytes or blockBytes
- we should also pass in maxGas or blockGas